src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -161,7 +161,7 @@ public PersistentDataStore() {
     List<Activity> activities = new ArrayList<>();
 
     // Retrieve all activities from the datastore.
-    Query query = new Query("chat-activities").addSort("creation_time", SortDirection.DESCENDING);
+    Query query = new Query("chat-activities").addSort("creation_time", SortDirection.ASCENDING);
```

Comment:
Will this show the activities with the oldest ones on top? If yes, it makes sense to show the latest ones first.

Reply:
It seems really counter-intuitive, but for some reason, ascending order puts the latest ones first.